backend/test/akvo/lumen/transformation_test.clj

```diff
@@ -296,13 +296,13 @@
                                             "onError" "fail"}))]
         (is (= tag ::lib/bad-request))))
 
-    (testing "Fail infinite loop"
+    (testing "Infinite loop is managed in js sandbox impl"
       (let [[tag _] (apply-transformation (derive-column-transform
                                            {"args" {"code" "while(true) {}"
```

Comment:
this test wasn't correct, the bad-request response was given thus parsing was failing instead of covering infinite loops (as org.javadelight/delight-nashorn-sandbox does)